docs/release-notes/8.0.31-23.md

```diff
@@ -0,0 +1,154 @@
+
+* MySQL adds support for the SQL standard [`INTERSECT`](https://dev.mysql.com/doc/refman/8.0/en/intersect.html) and [`EXCEPT`](https://dev.mysql.com/doc/refman/8.0/en/except.html) table operators.
+
+* InnoDB supports parallel index builds. This improves index build performance. The sorted index entries are loaded into a B-tree in a multithread. In previous releases, this action was performed by a single thread.
+
+* The Performance and sys schemas show metrics for the global and session memory limits introduced in MySQL 8.0.28.
+
+    The following columns have been added to the Performance Schema tables:
+
+    | Performance Schema tables                                                            | Columns                                                   |
+    | ------------------------------------------------------------------------------------ | --------------------------------------------------------- |
+    | SETUP_INSTRUMENTS                                                                    | FLAGS                                                     |
+    | THREADS                                                                              | CONTROLLED_MEMORY, MAX_CONTROLLED_MEMORY, TOTAL_MEMORY, MAX_TOTAL_MEMORY |
+    | EVENTS_STATEMENTS_CURRENT, EVENTS_STATEMENTS_HISTORY, EVENTS_STATEMENTS_HISTORY_LONG | MAX_CONTROLLED_MEMORY, MAX_TOTAL_MEMORY                   |
+    | Statement Summary Tables                                                             | MAX_CONTROLLED_MEMORY, MAX_TOTAL_MEMORY                   |
+    | Performance Schema Connection Tables                                                 | MAX_SESSION_CONTROLLED_MEMORY, MAX_SESSION_TOTAL_MEMORY   |
+    | PREPARED_STATEMENTS_INSTANCES                                                        | MAX_CONTROLLED_MEMORY, MAX_TOTAL_MEMORY                   |
+
+    The following columns have been added to the sys schema `STATEMENT_ANALYSIS` and `X$STATEMENT_ANALYSIS` views:
+
+    * MAX_CONTROLLED_MEMORY
+
+    * MAX_TOTAL_MEMORY
+
+    The `controlled_by_default` flag has been added to the `PROPERTIES` column of the `SETUP_INSTRUMENTS` table.
+
+    Now, you can add and remove non-global memory instruments to the set of controlled-memory instruments. To do this, set the value of the `FLAGS` column of `SETUP_INSTRUMENTS`.
+
+    ```sql
+    SQL> UPDATE PERFORMANCE_SCHEMA.SETUP_INTRUMENTS SET FLAGS="controlled" 
+    WHERE NAME='memory/sql/NET::buff';
+    ```
+
+* The `audit_log_flush` variable has been deprecated and will be removed in future releases.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.31 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-31.html).
+
+## Improvements
+
+* [PS-7963](https://jira.percona.com/browse/PS-7963): The trigger definitions are not cached in the table cache when using SELECT statements.
```

Comment:
@alina-derkach-oaza May be we should give a link to the new `table_open_cache_triggers` variable that this improvement introduces.

Reply:
as well as
```
table_open_cache_triggers_hits
table_open_cache_triggers_misses
table_open_cache_triggers_overflows
```
status variables

Reply:
updated